bin/ch/HostConfigFlagsList.h

```diff
@@ -4,7 +4,10 @@
 //-------------------------------------------------------------------------------------------------------
 
 #ifdef FLAG
-FLAG(BSTR, Serialized,                    "If source is UTF8, deserializes from bytecode file", NULL)
-FLAG(BSTR, GenerateLibraryByteCodeHeader, "Generate bytecode header file from library code", NULL)
+FLAG(BSTR, dbgbaseline,                     "Baseline file to compare debugger output", NULL)
+FLAG(BSTR, DebugLaunch,                     "Create the test debugger and execute test in the debug mode", NULL)
```

Comment:
Do you think the DebugLaunch requires to be BSTR, unless you want to have hybrid support on ch?